caddy/https/crypto_test.go

```diff
@@ -105,7 +105,12 @@ func PrivateKeyBytes(key crypto.PrivateKey) []byte {
 	case *rsa.PrivateKey:
 		keyBytes = x509.MarshalPKCS1PrivateKey(key)
 	case *ecdsa.PrivateKey:
-		keyBytes, _ = x509.MarshalECPrivateKey(key)
+		var err error
+		var t *testing.T
```

Comment:
What's up with this, isn't it nil? Why not use the real T value from the test function? @elcore @wmark